### PR TITLE
feat: use the latest client package

### PR DESCRIPTION
**Pull request by VanishMax**
Uses https://github.com/penumbra-zone/web/pull/1567 and https://github.com/prax-wallet/web/pull/134 – new API of the `client` package

### CODE REVIEW

File: app/fetchers.ts [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-const createFetchClient = (wallet: string) => {
-  return createPenumbraClient<typeof ViewService>(ViewService, wallet);
-};
+import { client } from './penumbra';
 
-export const fetchAddress = async (wallet: string, account: number): Promise<string | undefined> => {
-  const client = await createFetchClient(wallet);
-  const res = await client.addressByIndex({ addressIndex: { account } });
+export const fetchAddress = async (account: number): Promise<string | undefined> => {
+  const viewService = createServiceClient(client, ViewService);
```
Comment (grod220) [inline app/fetchers.ts]: Can we make the service client external to this function? Aka, is the approval side effect fixed?

Comment (VanishMax) [reply app/fetchers.ts]: Moved the `viewService` atop the fetchers, works fine 👍

**Comment by VanishMax** [comment]
@grod220 @turbocrime this PR is finalized and working, ready for review

**Comment by grod220** [comment]
There seems to be two issues in my testing:

1) After denial, it refuses to retry without a page refresh

https://github.com/user-attachments/assets/b487abb3-5a7b-47e2-b5c8-015c9d9523c5

2) After disconnecting, you cannot re-connect without a page refresh

https://github.com/user-attachments/assets/232f1148-36c9-495f-9c96-bdb98e67547a

**Review by grod220 — APPROVED** [review]

**Comment by grod220** [comment]
Discussed offline about following up with AI's

**Pull request merged**